lambda-deployment-orca/src/main/java/com/amazon/aws/spinnaker/plugin/lambda/utils/LambdaCloudDriverUtils.java

```diff
@@ -337,30 +335,30 @@ public String getCanonicalVersion(LambdaDefinition lf, String inputVersion, Stri
     }
 
     public List<String> getSortedRevisions(LambdaDefinition lf) {
-        List<String> revisions =  lf.getRevisions().values().stream().collect(Collectors.toList());
+        List<String> revisions = lf.getRevisions().values().stream().collect(Collectors.toList());
         List<Integer> revInt = revisions.stream()
-                                        .filter(x -> { return NumberUtils.isCreatable(x); })
-                                        .map(x -> { return Integer.valueOf(x); })
-                                        .collect(Collectors.toList());
+                .filter(x -> {
+                    return NumberUtils.isCreatable(x);
+                })
+                .map(x -> {
+                    return Integer.valueOf(x);
+                })
+                .collect(Collectors.toList());
```

Comment:
having them inline is actually crisper.

Reply:
Yeah that was auto formatting... need to get my inttellij updated ;)